### PR TITLE
[REVIEW] gpu/build: adding some dependencies

### PR DESCRIPTION
**Pull request by mario21ic**

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by dillon-cullinan** [comment]
add to whitelist

**Comment by williamBlazing** [comment]
rerun tests

**Comment by mario21ic** [comment]
rerun tests

**Comment by mario21ic** [comment]
return tests

**Comment by mario21ic** [comment]
rerun tests

**Comment by williamBlazing** [comment]
rerun tests

**Comment by mario21ic** [comment]
rerun tests

**Comment by mario21ic** [comment]
rerun tests

**Comment by mario21ic** [comment]
rerun tests

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -72,6 +75,9 @@ $WORKSPACE/build.sh
 if hasArg --skip-tests; then
     logger "Skipping Tests..."
 else
+    INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX}}}
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$INSTALL_PREFIX/lib
```
Comment (dillon-cullinan) [inline ci/gpu/build.sh]: I don't believe these need to be here. This else block is just running the tests, they should've already been installed previously with the build.sh command.

Comment (williamBlazing) [reply ci/gpu/build.sh]: the tests wont run properly without these environment variables

**Pull request merged**